lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -633,3 +637,95 @@ def test_error(self):
         self.assertEqual(response.status_code, 400)
         response_data = json.loads(response.content)
         self.assertEqual(response_data, expected_response_data)
+
+
+@httpretty.activate
+class CommentViewSetPartialUpdateTest(DiscussionAPIViewTestMixin, ModuleStoreTestCase):
+    """Tests for CommentViewSet partial_update"""
+    def setUp(self):
+        super(CommentViewSetPartialUpdateTest, self).setUp()
+        self.url = reverse("comment-detail", kwargs={"comment_id": "test_comment"})
+
+    def test_basic(self):
+        self.register_get_user_response(self.user)
+        cs_thread = make_minimal_cs_thread({
+            "id": "test_thread",
+            "course_id": unicode(self.course.id),
+        })
+        self.register_get_thread_response(cs_thread)
+        cs_comment = make_minimal_cs_comment({
+            "id": "test_comment",
+            "course_id": cs_thread["course_id"],
+            "thread_id": cs_thread["id"],
+            "username": self.user.username,
+            "user_id": str(self.user.id),
+            "created_at": "2015-06-03T00:00:00Z",
+            "updated_at": "2015-06-03T00:00:00Z",
+            "body": "Original body",
+        })
+        self.register_get_comment_response(cs_comment)
+        self.register_put_comment_response(cs_comment)
+        request_data = {"raw_body": "Edited body"}
+        expected_response_data = {
+            "id": "test_comment",
+            "thread_id": "test_thread",
+            "parent_id": None,  # TODO: we can't get this without retrieving from the thread :-(
```

Comment:
Just to verify, Is this TODO going to be addressed in future PRs for discussion API?